app/Http/Livewire/WepaySignup.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://opensource.org/licenses/AAL
+ */
+
+namespace App\Http\Livewire;
+
+use App\Models\Company;
+use App\Models\User;
+use Illuminate\Support\Facades\Hash;
+use Livewire\Component;
+
+class WepaySignup extends Component
+{
+    public $user;
+    public $user_id;
+    public $company_key;
+    public $first_name;
+    public $last_name;
+    public $email;
+
+    public $terms;
+    public $privacy_policy;
+
+    public $saved;
+
+    protected $rules = [
+        'first_name' => ['sometimes'],
+        'last_name' => ['sometimes'],
+        'email' => ['required', 'email'],
+    ];
+
+    public function mount()
+    {
+        $user = User::find($this->user_id);
```

Comment:
It is possible to pass the user entity as well as company's here so it doesn't need additional query.